packages/baby-jubjub/src/baby-jubjub.ts

```diff
@@ -75,24 +76,14 @@ export function addPoint(p1: Point<bigint>, p2: Point<bigint>): Point<bigint> {
  * @returns The resulting point representing the public key.
  */
 export function mulPointEscalar(base: Point<bigint>, e: bigint): Point<bigint> {
-    if (scalar.isZero(e)) {
-        return [BigInt(0), BigInt(1)]
-    }
-    const eBits: Array<boolean> = []
-    while (!scalar.isZero(e)) {
-        if (scalar.isOdd(e)) {
-            eBits.push(true)
-        } else {
-            eBits.push(false)
-        }
-        e = scalar.shiftRight(e, BigInt(1))
-    }
+    e %= order
 
-    let R0: Point<bigint> = base
-    let R1: Point<bigint> = addPoint(base, base)
+    let R0: Point<bigint> = [0n, 1n]
```

Comment:
I see there's a unit test covering multiply by zero, which is good to confirm this still behaves the same way as before without the special-case.
As a side suggestion, it might be clearer to make this a public const `id` like is used in the unit tests, to make it clear that [0n, 1n] is the representation of the zero/id point in this curve.

Reply:
I like that. It is done.